src/test/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatusTest.java

```diff
@@ -107,7 +107,7 @@ public void testUseDefaultContext() throws Exception {
         given(ghprbPullRequest.isMergeable()).willReturn(false);
 
         GhprbSimpleStatus globalStatus =
-                new GhprbSimpleStatus(true, context, statusUrl, "test1", "test2", false, false, new ArrayList<GhprbBuildResultMessage>(0));
+                new GhprbSimpleStatus(true, context, statusUrl, "test1", "test2", false, null, new ArrayList<GhprbBuildResultMessage>(0));
```

Comment:
I'm not 100% sure, but I'm pretty sure you can add a test file with a workspace mock that will enable you to read in the test file and check you get the right number. We should test some of the logic behind that. Maybe just mock out what does the reading. Then test that you get a number back if someone does give a valid string.